test/cases/parsing/typeof/index.js

```diff
@@ -29,13 +29,6 @@ it("should answer typeof require.ensure correctly", function() {
 it("should answer typeof require.resolve correctly", function() {
 	(typeof require.resolve).should.be.eql("function");
 });
-it("should answer typeof System correctly", function() {
```

Comment:
These two tests should be moved into `configCases/parsing/system.import`

Reply:
done